integration-tests/tests/link-to-new-workspace.js

```diff
@@ -21,6 +21,8 @@ const testLinkToNewWorkspaceFn = withUserToken(async ({ testUrl, page, token })
     await click(page, clickable({ text: `${newWorkspaceBillingAccount}` }))
     await click(page, clickable({ text: 'Create Workspace' }))
     await waitForNoSpinners(page)
```

Comment:
When running this test through the flake checker I was seeing I am seeing some (4/200) of these errors as well:

```
TimeoutError: waiting for XPath `//*[@data-icon="loadingSpinner"]` to be hidden failed: timeout 30000ms exceeded
    at new WaitTask (/Users/psantos/projects/terra-ui/integration-tests/node_modules/puppeteer/src/common/DOMWorld.ts:798:28)
    at DOMWorld.waitForXPath (/Users/psantos/projects/terra-ui/integration-tests/node_modules/puppeteer/src/common/DOMWorld.ts:695:22)
    at Frame.waitForXPath (/Users/psantos/projects/terra-ui/integration-tests/node_modules/puppeteer/src/common/FrameManager.ts:1304:47)
    at Page.waitForXPath (/Users/psantos/projects/terra-ui/integration-tests/node_modules/puppeteer/src/common/Page.ts:3269:29)
    at waitForNoSpinners (/Users/psantos/projects/terra-ui/integration-tests/utils/integration-utils.js:123:15)
    at /Users/psantos/projects/terra-ui/integration-tests/tests/link-to-new-workspace.js:23:11
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at /Users/psantos/projects/terra-ui/integration-tests/jest/jest-utils.js:75:24
```

I ran it with this code and I did not see the issue in 200 runs (this is a new util I added last week to address flaky spinner failures):
```suggestion
  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Link to a workspace' })) })
```